integration_tests/tests/waf-regional.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg(feature = "waf-regional")]
+
+extern crate rusoto_core;
+extern crate rusoto_waf_regional;
+
+use rusoto_waf_regional::{WAFRegional, WAFRegionalClient, ListRulesRequest};
+use rusoto_core::{DefaultCredentialsProvider, Region};
+use rusoto_core::default_tls_client;
+
+#[test]
+fn should_list_rules() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = WAFRegionalClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
+    let request = ListRulesRequest::default();
+
+    let result = client.list_rules(&request).unwrap();
```

Comment:
I'd love to see future integration tests to use `expect` instead of `unwrap`.  It'll make regressions easier to find: it took a little bit of sleuthing to find the failing test in S3 [in another PR](https://github.com/rusoto/rusoto/pull/666#issuecomment-307286032).  If it explicitly declares what failed it'd be easier to narrow things down, especially as we get more tests.

In this case how about `client.list_rules(&request).expect("waf regional list_rules call errored");` ?